scripts/release/verify-release.mjs

```
@@ -0,0 +1,110 @@
+const ROOT = process.cwd();
+const RELEASE_DIR = path.join(ROOT, 'release');
+const EVIDENCE_DIR = path.join(ROOT, '.repoos', 'evidence');
+
+function sha256File(filePath) {
+  const input = readFileSync(filePath);
+  return createHash('sha256').update(input).digest('hex');
+}
+
+function collectTimestampKeys(value, current = '$', findings = []) {
+  if (Array.isArray(value)) {
+    value.forEach((entry, index) => collectTimestampKeys(entry, `${current}[${index}]`, findings));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)